### PR TITLE
Reactivate scada test

### PR DESCRIPTION
**Pull request by JoranAngevaare**
Reactivate scada

### CODE REVIEW

**Review by jmosbacher — APPROVED** [review]
Thanks Joran!

File: tests/test_scada.py [hunk 1]
```diff
@@ -97,9 +93,6 @@ def resources_available(self):
         """
         if not straxen.utilix_is_configured('scada','scdata_url',):
             self.skipTest("Cannot test scada since we have no access to xenon secrets.)")
-        
-        try:
-            self.sc = straxen.SCADAInterface(use_progress_bar=False)
-            self.sc.get_new_token()
-        except requests.exceptions.SSLError:
-            self.skipTest("Cannot reach database since HTTPs certifcate expired.")
```
Comment (WenzDaniel) [inline tests/test_scada.py:105]: I think we should keep it in case the certificate expires.

**Review by WenzDaniel — APPROVED** [review]
Thanks

**Comment by JoranAngevaare** [comment]
Thanks

**Pull request merged**